toolchain/sem_ir/inst_namer.cpp

```diff
@@ -1129,6 +1143,26 @@ auto InstNamer::NamingContext::NameInst() -> void {
       AddInstName("require_complete");
       return;
     }
+    case CARBON_KIND(RequireDecl inst): {
+      AddInstName("require");
+
+      const auto& require = sem_ir().require_impls().Get(inst.require_impls_id);
+      auto parent_scope_inst = sem_ir().insts().Get(
+          sem_ir().name_scopes().Get(require.parent_scope_id).inst_id());
+      auto scope_id = ScopeId::None;
+      if (auto iface_decl = parent_scope_inst.TryAs<SemIR::InterfaceDecl>()) {
+        scope_id = inst_namer_->GetScopeFor(iface_decl->interface_id);
+      } else {
+        auto con_decl = parent_scope_inst.As<SemIR::NamedConstraintDecl>();
+        scope_id = inst_namer_->GetScopeFor(con_decl.named_constraint_id);
+      }
+      // FIXME: Do this?
+      // AddBlockLabel(scope_id, inst.decl_block_id, "require",
+      //                            SemIR::LocId(inst_id_));
```

Comment:
Missed cleaning this up? IIRC this is mainly for if we start having things like conditions within blocks, e.g. supporting `if x then y else z` which is currently allowed in statement expressions but not declarations (but probably should work more broadly). So you may not observe it now, but similar to `class` adding a label, my guess is it's probably for the better to not get lost later.

```suggestion
      AddBlockLabel(scope_id, inst.decl_block_id, "require",
                                 SemIR::LocId(inst_id_));
```

Reply:
Ah okay, thanks. Will do.